server/protocols/webadmin/webadmin-mailqueue/src/main/java/org/apache/james/webadmin/routes/MailQueueRoutes.java

```diff
@@ -238,4 +251,107 @@ private boolean filter(MailQueueItemDTO item, Optional<Boolean> isDelayed) {
         return isDelayed.map(delayed -> delayed == item.getNextDelivery().isPresent())
             .orElse(true);
     }
+
+    @DELETE
+    @Path("/{mailQueueName}/mails")
+    @ApiImplicitParams({
+        @ApiImplicitParam(required = true, dataType = "string", name = "mailQueueName", paramType = "path"),
+        @ApiImplicitParam(
+                required = false, 
+                dataType = "MailAddress", 
+                name = SENDER_QUERY_PARAM, 
+                paramType = "query",
+                example = "?sender=sender@james.org",
+                value = "The sender of the mails to be deleted should be equals to this query parameter."),
+        @ApiImplicitParam(
+                required = false, 
+                dataType = "String", 
+                name = NAME_QUERY_PARAM,
+                paramType = "query",
+                example = "?name=mailName",
+                value = "The name of the mails to be deleted should be equals to this query parameter."),
+        @ApiImplicitParam(
+                required = false, 
+                dataType = "MailAddress", 
+                name = RECIPIENT_QUERY_PARAM, 
+                paramType = "query",
+                example = "?recipient=recipient@james.org",
```

Comment:
why do you include the `?`?

Reply:
Because I did the same I imagine....

Reply:
We have done the same thing everywhere, I just add that for consistency.